screens/Login.js

```diff
@@ -1,126 +1,93 @@
 import React from 'react';
-import { StyleSheet, Text, View, Dimensions } from 'react-native';
-import { Font } from 'expo';
+import { StyleSheet, Text, View, Dimensions, ImageBackground} from 'react-native';
 
 const window = Dimensions.get("window")
 
 export default class Login extends React.Component {
 
   render() {
     return (
-      <View style={styles.container}>
-        <View style={styles.topBanner} />
-        <Text style={styles.loginText}>Login</Text>
-        <View style={styles.loginBox} />
-        <Text style={styles.usernameText}>Username</Text>
-        <View style={styles.usernameLine} />
-        <Text style={styles.passwordText}>Password</Text>
-        <View style={styles.passwordLine} />
-        <View style={styles.submitButton} />
-        <Text style={styles.submitText}>Submit!</Text>
-        <Text style={styles.forgotText}>Forgot Password?</Text>
-      </View>
+      <ImageBackground style={styles.container}
+          source={require('./../assets/images/background.png')}>
+            <View style={{justifyContent: 'center', alignItems: 'center'}}>
+              <Text style={styles.loginText}>Login</Text>
+              <View style={styles.loginBox}>
+                  <Text style={styles.usernameText}>Username</Text>
```

Comment:
This needs to be changed to `TextInput`. Start thinking about practical functionality